stateproof/builder.go

```diff
@@ -105,7 +128,7 @@ func (spw *Worker) loadSignaturesIntoBuilder(buildr *builder) error {
 	return nil
 }
 
-func (spw *Worker) createBuilder(rnd basics.Round) (builder, error) {
+func (spw *Worker) createAndPersistBuilder(rnd basics.Round, votersFetcher ledgercore.VotersForRoundFetcher) (builder, error) {
```

Comment:
now `createAndPersistBuilder` might returns an error in case `voters == nil` and fails the commit.
Since there are cases where `voters == nil`  (when state proof kicks-in). We should consider not treating it as an error.

Reply:
Let's decide whether we should fail the commit at all, and then I'll change this.

Reply:
Still need to figure out what to do where round % 256 == 0 and we have no commitment.